### PR TITLE
[SDL 0253] Feature/New vehicle data StabilityControlsStatus

### LINKED ISSUE #3086: [SDL 0253] New vehicle data StabilityControlsStatus
Proposal: [New vehicle data StabilityControlsStatus](https://github.com/smartdevicelink/sdl_evolution/edit/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md)

>This proposal is to add new vehicle data `StabilityControlsStatus` to know if stability controls like ESC, Traction Control etc. are active.

Review: https://github.com/smartdevicelink/sdl_evolution/issues/841

### PR DESCRIPTION
**Pull request by SNiukalov**
Fixes #3086 

This PR is **ready** for review.

### Risk
This PR makes **minor** API changes.

### Testing Plan
ATF scripts.

Use scripts from:
[PR#2421](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2421)

### Summary
PR contains sdl_core implementation of [SDL-0253] New vehicle data StabilityControlsStatus

##### Breaking Changes
* [rpc_spec #233](https://github.com/smartdevicelink/rpc_spec/pull/233)

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/smartdevicelink/jsoncpp.git
 [submodule "tools/rpc_spec"]
 	path = tools/rpc_spec
-	url = https://github.com/smartdevicelink/rpc_spec.git
+	url = https://github.com/SNiukalov/rpc_spec.git
```
Comment (LuxoftAKutsan) [inline .gitmodules:6]: Can we move to LuxoftSDL?

**Comment by SNiukalov** [comment]
NOTE: Please be informed, this PR contains the temporary commits. that should be removed before merge:
3358d5eae8bfadf5275907e1eb7dd5e3fed249d1 - temporary commit for rpc_spec repository.
57fdfe081b780c3697499541123a83f447e28434 - temporary commit for rpc_spec repository.

File: src/appMain/sdl_preloaded_pt.json [hunk 2]
```diff
@@ -3907,6 +3911,13 @@
                     "mandatory": false,
                     "since": "5.0"
                 },
+                {
+                    "name": "stabilityControlsStatus",
```
Comment (IGapchuk) [inline src/appMain/sdl_preloaded_pt.json:3915]: @IGapchuk move it to a separate commit with comment that it redundant

**Comment by LitvinenkoIra** [comment]
@atiwari9 please review this PR

**Comment by atiwari9** [comment]
PR Approved.

Validated with ATF scripts: [a406f7d](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2421/commits/a406f7d1ca1b1376e1739906b28bb80fc9af2847)


Logs: 
[Test_Results_StabilityControlsStatus_a406f7d_2.zip](https://github.com/smartdevicelink/sdl_atf_test_scripts/files/4794887/Test_Results_StabilityControlsStatus_a406f7d_2.zip)

**Comment by LitvinenkoIra** [comment]
@theresalech PR is ready for Livio review. Thank you!

**Comment by ShobhitAd** [comment]
@LitvinenkoIra The RPC spec PR for this feature https://github.com/smartdevicelink/rpc_spec/pull/233 has been merged into develop. Please update the `.gitmodules` file to use the `smartdevicelink` remote and update the `tools/rpc_spec` submodule commit. Thanks

**Comment by LitvinenkoIra** [comment]
@ShobhitAd Thanks, please check the https://github.com/smartdevicelink/sdl_core/pull/3304/commits/aab504b3943f100183f56bdbb109eed974a26487
